pages/glossary/disabled-element.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Disabled Element
+key: disabled-element
+---
+
+An element that is normally [perceivable][] and [operable][], but has been rendered [inoperable][operable] either temporarily or permanently using one of the following properties, is considered disabled:
+
+1. The [`disabled`][disabled] attribute. The presence of this attribute, regardless of its value, on a
+
+   - [`button`][button],
+   - [`input`][input],
+   - [`select`][select],
+   - [`textarea`][textarea],
+   - [form-associated custom element][],
+   - or [`fieldset`][fieldset] element
+
+   will disable the element itself if it is not a [`fieldset`][fieldset] or, if it is, will disable any [descendants][descendant] of the element, excluding those that are [descendants][descendant] of the first [`legend`][legend] [child][] of the element.
+
+   **Note:** When the [`disabled`][disabled] attribute is specified on a [`fieldset`][fieldset] element, [shadow-including descendants][shadow-including descendant] are **not** disabled by default. Such behavior may however be explicitly implemented by [form-associated custom elements][form-associated custom element].
+
+2. The [`aria-disabled`][aria-disabled] attribute. The presence of this attribute with a value that is an [ASCII case-insensitive][] match for `true` will semantically disable an element and its [focusable][] [shadow-including descendants][shadow-including descendant].
```

Comment:
I think a definition for "semantically disable" is needed. Also might be a good idea to define "functionally disable" used in the note.